schematic/models/validate_attribute.py

```diff
@@ -356,7 +356,7 @@ class ValidateAttribute(object):
         - Add string length validator
     """
 
-    def get_target_manifests(target_component):
+    def get_target_manifests(target_component, project_scope: List):
```

Comment:
@GiaJordan are we guaranteed we have a project_scope set here? I.e. do we need a default value?

Reply:
As it stands, yes. This is a new method I made that currently is only called from the `cross_validation` method as part of the validation process where `project_scope `has either been set to a list or initialized to `None`. However, if we think that this method has utility outside cross manifest validation and want to move it outside the `ValidateAttribute` object so it can be used elsewhere then I should initialize the scope to `none `again here

Reply:
Thanks - great! Then we can keep it strictly required for now.